srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -81,5 +92,5 @@ def render(self, request: Request):
 def handler(**kwargs):
     prepare_database()
     ports = kwargs.get('ports', default_ports)
-    for port in ports:
-        endpoints.serverFromString(reactor, f'tcp:{port}').listen(server.Site(HTTP()))
+    reactor.callInThread(http_endpoint(ports))
+    reactor.callInThread(https_endpoint)
```

Comment:
Why is one being called with ports and the other not being called at all? Also how does the callInThread work here?